jbake-core/src/main/java/org/jbake/app/configuration/ConfigUtil.java

```diff
@@ -56,4 +58,40 @@ public JBakeConfiguration loadConfig(File source) throws ConfigurationException
         CompositeConfiguration configuration = load(source);
         return new DefaultJBakeConfiguration(source, configuration);
     }
+
+     /**
+     * Given a file inside content it return
+     * the relative path to get to the root.
+     * 
+     * Example: /content and /content/tags/blog will return '../..'
+     * 
+     * @param sourceFile the file to calculate relative path for
+     * @return
+     */
+    static public String getPathToRoot(JBakeConfiguration config, File rootPath, File sourceFile) {
+
+        Path r = Paths.get(rootPath.toURI());
+        Path s = Paths.get(sourceFile.getParentFile().toURI());
+        Path relativePath = s.relativize(r);
+
+        StringBuilder sb = new StringBuilder();
+
+        sb.append(relativePath.toString());
+
+        if (config.getUriWithoutExtension()) {
+            sb.append("/..");
+        }
+        if(sb.length()>0) {  // added as calling logic assumes / at end.
+            sb.append("/");
+        }
+        return sb.toString();
+    }
+
+    static public String getPathtoDestinationRoot(JBakeConfiguration config, File sourceFile) {
```

Comment:
Little typo `getPathtoDestinationRoot` -> `getUriPathToDestinationRoot`. What do you think?